### PR TITLE
add new project with menu food name

### PR DESCRIPTION
**Pull request by moloudii**

### CODE REVIEW

File: index-menu-food.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        <meta name="apple-mobile-web-app-capable" content="yes"> 
+        <!-- <link rel="icon" type="image/png" href="img/favicon.ico"> -->
+        <link rel="apple-touch-icon-precomposed" sizes="144x144" href="img/apple-touch-icon-144x144.png">
+        <link rel="apple-touch-icon-precomposed" sizes="72x72" href="img/apple-touch-icon-72x72.png"> 
+
+        <link rel="stylesheet" href="css/style.css" media="all">
+
+    </head>
+    <body>
+        <div class="picture-background">
```
Comment (Mekaeil) [inline index-menu-food.html]: Wrap all of the tags into the section with class = 'menu-widget--item

File: index-menu-food.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                </div>
+                    <div class="title-menu">
+                        <p class="p-title-menu">labore nostrum dicta<br> earum debitis debitis</p> 
+                    </div>
+                <div class="left-menu">
+                    <div class="name-food">
+                        <p class="bolder">labore nostrum dicta</p>
+                        <p class="food-contents">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia id impedit illo optio </p>
+                    </div>
+                    <div class="price">$12.33</div>
```
Comment (Mekaeil) [inline index-menu-food.html]: you should wrap each item in A tag and price for each item should in it. 
```
<article class='menu--item'>
                    <div class="name">
                        <p class="bolder">labore nostrum dicta</p>
                        <p class="contents">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia id impedit illo optio </p>
                    </div>
                    <div class="price">$12.33</div>
</article>
```

File: index-menu-food.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                </div>
+                <div class="button">
+                    <button type="button">View full menu</button>
+                </div>
+            </div>
+        </div>
+
+        <br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
```
Comment (Mekaeil) [inline index-menu-food.html]: just 3 br is enough!

File: sass/modules/item/_menu-food.scss [hunk 4]
```diff
@@ -0,0 +1,418 @@
+/*========================================================================================
+*********    
+*******      
+*****       DIV ELEMENT FOOD MENU
+***         
+*/
+    .picture-background{
+        background-image    : url("../img/menu-food/bg-menu-food.jpg");
```
Comment (Mekaeil) [inline sass/modules/item/_menu-food.scss]: In `img` folder create a directory called it `item' and add all of the images in it.

File: sass/modules/item/_menu-food.scss [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+
+    .button{
+        width: 100%;
+        text-align          : center;
+        padding-top         : 20px;
+        display             : block;
+
+    }
+
+    .button button{
```
Comment (Mekaeil) [inline sass/modules/item/_menu-food.scss]: :/ why!

File: sass/modules/item/_menu-food.scss [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+
+            font-size       : 1.4em;
+            line-height     : 1.4em;
+
+        }
+
+        .price{
+
+            font-size       : 1.8em;
+            line-height     : 4.4em;
```
Comment (Mekaeil) [inline sass/modules/item/_menu-food.scss]: It's not a good way. please think about it and use a better way for it.

File: sass/modules/item/_menu-food.scss [hunk 7]
```diff
@@ -0,0 +1,418 @@
...
+
+            width           : 60%;
+            position        : absolute;
+            top             : 2em;
+            left            : 20%;
+
+        }
+
+        .name-food{
+
```
Comment (Mekaeil) [inline sass/modules/item/_menu-food.scss]: please remove extra space!

File: sass/modules/item/_menu-food.scss [hunk 8]
```diff
@@ -0,0 +1,418 @@
...
+        .left-menu{
+
+            width           : 50%;
+            float           : left;
+
+        }
+
+        .right-menu{
+
```
Comment (Mekaeil) [inline sass/modules/item/_menu-food.scss]: remove extra space in all of the file!

File: index-menu-food.html (hunk 3 above)
Comment (moloudii) [reply index-menu-food.html]: Not
Because Persian class is not performing well.

File: sass/modules/item/_menu-food.scss (hunk 6 above)
Comment (moloudii) [reply sass/modules/item/_menu-food.scss]: I didn't understand what to change

**Pull request merged**